includes/FormFactory/ManageWikiFormFactoryBuilder.php

```diff
@@ -197,15 +197,12 @@ private static function buildDescriptorCore(
 		}
 
 		if ( ExtensionRegistry::getInstance()->isLoaded( 'WikiDiscover' ) && $config->get( 'WikiDiscoverUseDescriptions' ) ) {
-			$mwSettings = new ManageWikiSettings( $dbName );
-			$setList = $mwSettings->list();
-
 			$formDescriptor['description'] = [
 				'label-message' => 'managewiki-label-description',
 				'type' => 'text',
-				'default' => $setList['wgWikiDiscoverDescription'] ?? '',
+				'default' => $remoteWiki->getExtraFieldData( 'description' ) ?? '',
 				'maxlength' => 512,
-				'disabled' => !$ceMW,
+				'disabled' => true,
```

Comment:
NOTE: We just disable for now as this will be moved to WikiDiscover with hooks afterwards and we want it to be disabled during migration.